nagios/stackstate_checks/nagios/data/conf.yaml.example

```diff
@@ -0,0 +1,9 @@
+init_config:
+  min_collection_interval: 30
```

Comment:
in agent v2 this config is part of instance https://docs.datadoghq.com/developers/write_agent_check/?tab=agentv6v7#collection-interval

I would remove it

Reply:
Removed